src/testing/reduced_order_pod_adaptation.h

```diff
@@ -0,0 +1,70 @@
+#ifndef __BURGERS_REWIENSKI_ROM_H__
+#define __BURGERS_REWIENSKI_ROM_H__
+
+#include "tests.h"
+#include "dg/dg.h"
+#include "parameters/all_parameters.h"
+#include "functional/functional.h"
+
+namespace PHiLiP {
+namespace Tests {
+
+/// Burgers Rewienski snapshot
+template <int dim, int nstate>
+class ReducedOrderPODAdaptation: public TestsBase
+{
+public:
+    /// Constructor.
+    ReducedOrderPODAdaptation(const Parameters::AllParameters *const parameters_input);
+
+    /// Run test
+    int run_test () const override;
+};
+
+///Functional to take the integral of the solution
+template <int dim, int nstate, typename real>
+class BurgersRewienskiFunctional : public Functional<dim, nstate, real>
```

Comment:
It could be used for other equations, but it just takes the integral of the solution, so it's not very interesting. Since it's not used anywhere else, I figured it might be best to put it in the test not to clutter the functional directory with a functional that's only used once.

Reply:
Sure, if it's only used for this test.

Reply:
Sure, if it's only used for this test.